### PR TITLE
[Chat][Storybook] Updated wording

### PR DESCRIPTION
**Pull request by jpeng-ms**
# What

Updated wording based on comments received from PR review team: https://github.com/MicrosoftDocs/azure-docs/pull/108602/files

# Why

consistent

# How Tested

storybook link

# Process & policy checklist

- [x] I have updated the project documentation to reflect my changes if necessary.
- [x] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: packages/storybook/stories/Examples/TeamsInterop/TeamsInteropDocs.tsx [hunk 1]
```diff
@@ -32,12 +32,13 @@ export const getDocs: () => JSX.Element = () => {
...
+      <Description>- Drag & drop into the chat area.</Description>
+      <Description>- Upload an image as a file via the "Upload from this device" button.</Description>
       <Description>
-        Inline images are images that are copied and pasted directly into the send box of Teams client. For images that
-        were uploaded via "Upload from this device" menu or via drag-and-drop (such as dragging images directly to the
-        send box) in Teams, they are not supported at this time. To copy an image, the Teams user can either use
-        operating system's context menu to copy the image file then paste it into the send box of their Teams client, or
-        use keyboard shortcuts instead.
+        Azure Communication Services currently support only option one for copying and pasting the image. Users can
+        achieve it keyboard shortcuts or the operating system's context menu options for copy and paste.
```
Comment (JoshuaLai) [inline packages/storybook/stories/Examples/TeamsInterop/TeamsInteropDocs.tsx:41]: I don't think this sentence makes sense. Did you mean to say:
`Users can achieve this by using keyboard shortcuts or the operating system's context menu options for copy and paste.`

File: packages/storybook/stories/Examples/TeamsInterop/TeamsInteropDocs.tsx [hunk 2]
```diff
@@ -32,12 +32,13 @@ export const getDocs: () => JSX.Element = () => {
       <Source code={LobbyComponentText} />
       <Heading>Inline Images</Heading>
       <SingleLineBetaBanner />
+      <Description>Microsoft Teams allows users to share images by:</Description>
+      <Description>- Copying & paste into the box at the bottom of the chat.</Description>
```
Comment (jimchou-dev) [inline packages/storybook/stories/Examples/TeamsInterop/TeamsInteropDocs.tsx:36]: Change "box at the bottom" to "input box"?

File: packages/storybook/stories/Examples/TeamsInterop/TeamsInteropDocs.tsx [hunk 3]
```diff
@@ -32,12 +32,13 @@ export const getDocs: () => JSX.Element = () => {
...
+      <Description>- Drag & drop into the chat area.</Description>
```
Comment (jimchou-dev) [inline packages/storybook/stories/Examples/TeamsInterop/TeamsInteropDocs.tsx:37]: Change this to "Drag & drop into the input box area of the chat"?

File: packages/storybook/stories/Examples/TeamsInterop/TeamsInteropDocs.tsx (hunk 2 above)
Comment (jpeng-ms) [reply packages/storybook/stories/Examples/TeamsInterop/TeamsInteropDocs.tsx:36]: it's a very specific wording copied from Teams website

File: packages/storybook/stories/Examples/TeamsInterop/TeamsInteropDocs.tsx (hunk 3 above)
Comment (jpeng-ms) [reply packages/storybook/stories/Examples/TeamsInterop/TeamsInteropDocs.tsx:37]: i am not sure if input box is too technical, but the wording is from PR review team

**Review by JoshuaLai — APPROVED** [review]

**Review by jimchou-dev — APPROVED** [review]

**Pull request merged**